src/less/button/base/button.less

```diff
@@ -50,44 +50,6 @@ svg.fake-btn__icon {
     .btn-fixed-height-icon-dimensions;
 }
 
-button.btn--regular,
```

Comment:
I know I said we can brave with our breaking changes, but I'm a little bit hesitant about completely removing this regular button from the code right now. The different sizes I'm definitely happy to remove (as discussed offline), but I think this regular button might need to linger around a little longer with a deprecation notice. @seangates what do you think?

Reply:
Is "regular" an actual button style, like "primary".   I don't see it in the DS6.5 style guide at all.

Reply:
Let's keep it, but mark it as `deprecated`.